oci/private/tarball.bzl

```diff
@@ -49,13 +49,16 @@ def _tarball_impl(ctx):
     image = ctx.file.image
     tarball = ctx.actions.declare_file("{}/tarball.tar".format(ctx.label.name))
     yq_bin = ctx.toolchains["@aspect_bazel_lib//lib:yq_toolchain_type"].yqinfo.bin
+    bsdtar = ctx.toolchains["@aspect_bazel_lib//lib:tar_toolchain_type"]
     executable = ctx.actions.declare_file("{}/tarball.sh".format(ctx.label.name))
+    mtree = ctx.actions.declare_file("_{}.mtree".format(ctx.label.name))
     repo_tags = ctx.file.repo_tags
 
     substitutions = {
         "{{yq}}": yq_bin.path,
+        "{{tar}}": bsdtar.tarinfo.binary.path,
```

Comment:
Do you still need this substitution?

Reply:
yeah, sorry this was a broken attempt (putting it in two actions doesn't work since the artifacts produced by the first weren't plumbed to the second)

PTAL